common/script-variable-generator/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it would be nice to have something where modules stand out from the rest, so at least one intermediary level for common stuff